src/component/templete/shelter/AssignShelterInformation.js

```diff
@@ -11,8 +11,7 @@ import InputWithSelect from 'Molecules/input/InputWithSelect';
 import {initial_number, email_supplier} from 'Root/config/dummyDate_json';
 import InputWithEmail from 'Molecules/input/InputWithEmail';
 import DatePicker from 'Molecules/select/DatePicker';
-import {stagebar_style} from 'Organism/style_organism';
-
+import {stagebar_style} from 'Root/component/organism/style_organism copy';
```

Comment:
import 시에 상대참조보다 절대참조로 가도록 변경 'Organism/style_organism copy'로 변경해야함
Root/component/organism/**보다 Organism/**으로 경로를 단순화하도록 컴파일러에 설정되어 있음
'../'의 상대경로보다 절대경로로 import의 경로를 잡는것이 '../../..'과 같이 가독성을 떨어뜨리지 않고 리펙토링시
작업이 간편해짐
```suggestion
import {stagebar_style} from 'Organism/style_organism copy';
```